spec/requests/api/v0/tags_spec.rb

```diff
@@ -16,11 +16,12 @@
       get api_tags_path
 
       response_tag = response.parsed_body.first
-      expect(response_tag.keys).to match_array(%w[id name bg_color_hex text_color_hex])
+      expect(response_tag.keys).to match_array(%w[id name bg_color_hex text_color_hex short_summary badge])
       expect(response_tag["id"]).to eq(tag.id)
       expect(response_tag["name"]).to eq(tag.name)
       expect(response_tag["bg_color_hex"]).to eq(tag.bg_color_hex)
       expect(response_tag["text_color_hex"]).to eq(tag.text_color_hex)
+      expect(response_tag["short_summary"]).to eq(tag.short_summary)
```

Comment:
What does the "badge" value look like?  That appears untested.

Reply:
updated!